streamlit-python/README.md

```diff
@@ -54,14 +53,14 @@ This directory contains code organization by functionality, such as one distinct
 ```
 
 ## `snowflake.yml.jinja`
-While this file exists as a Jinja template, it is the only file that will be automatically rendered as a `snowflake.yml` file by the `snow app init` command, as described in the [README.md](../README.md). `snowflake.yml` is used by the snowCLI tool to discover your project's code and interact with snowflake with all relevant permissions and grants. Below are some of the configuration keys that you can use in `snowflake.yml` file.
+While this file exists as a Jinja template, it is the only file that will be automatically rendered as a `snowflake.yml` file by the `snow app init` command, as described in the [README.md](../README.md). `snowflake.yml` is used by Snowflake CLI to discover your project's code and interact with snowflake with all relevant permissions and grants. 
```

Comment:
```suggestion
While this file exists as a Jinja template, it is the only file that is automatically rendered as a `snowflake.yml` file by the `snow app init` command, as described in the [README.md](../README.md). Snowflake CLI uses the `snowflake.yml` file  to discover your project's code and interact with Snowflake using all relevant privileges and grants. 
```

Reply:
A few tech writing nits (avoid future tense and passive voice)

Also FYI, we are supposed to refer to "privileges" rather than "permissions."